nl_server/__init__.py

```diff
@@ -14,10 +14,43 @@
 
 from flask import Flask
 
+import logging
```

Comment:
@shifucun what do we need to do to enable logging? Right now, the logging.info statements appear to do nothing.

Reply:
It should work. I can take a look after this is checked in.

Reply:
ok thanks